manual/vcredist2008/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,28 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName         = 'vcredist2008'
+$softwareNamePattern = 'Microsoft Visual C++ 2008 Redistributable*'
+
+[array] $key = Get-UninstallRegistryKey $softwareNamePattern
+if ($key.Count -eq 1) {
```

Comment:
> Worked OK on AV ?

Not really, as can be seen from all the warnings generated on appveyor.
We would still get warnings if it's changed to 2 though, as there seem to be another version of C++ redist 2008 installed.